themes/bootprint3/less/bootprint.less

```diff
@@ -167,3 +167,34 @@ ul.random.image li img { margin: 0 auto; }
 }
 .jstree-facet .jstree-ocl::before { margin-left: -5px; }
 .jstree-node.active { background-color: @active-orange; }
+
+/* --- extended holdings --- */
+
```

Comment:
These styles seem identical to the ones in bootstrap.less. Bootprint will inherit from Bootstrap, so you don't need it in both places.